snapshot/lib/snapshot/collector.rb

```diff
@@ -97,7 +97,9 @@ def self.find_screenshot(activity)
         return find_screenshot(subactivity) if subactivity["Title"] == "Synthesize event"
       end
 
-      if activity["Attachments"]
+      if activity["Attachments"] && activity["Attachments"].last["Filename"]
```

Comment:
Uhhhh, this line might crash when `activity["Attachments"]` is an empty array, right?